src/tasks/daily/daily_shop_mixin.py

```
@@ -140,7 +149,11 @@ def buy_once(self, sum_credit):
         candidates.extend((item, False) for item in normal_results)
         candidates.extend((item, True) for item in (discount_results or []))
         for idx, (item, is_discount_item) in enumerate(candidates, start=1):
-            item_name = getattr(item, "name", None) or f"未知商品#{idx}"
+            # 模板标签（weapon_quota/orobertyl）替换为官方中文名，保证日志可读；
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)